test/minikube/minikube_fsfreeze_test.go

```diff
@@ -241,3 +249,123 @@ func TestFsFreezeBackupHook(t *testing.T) {
 		require.Equal(t, "Backup ID file /mnt/archive/nuodb/demo/backup.txt exists. Execute post-backup hook to complete current backup.", response.Message)
 	})
 }
+
+func TestHotSnapBackupHook(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+
+	// Create admin release
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &helm.Options{}, 1, "")
+
+	// Create database release with CSI driver storage class and backup hooks enabled
+	admin := fmt.Sprintf("%s-nuodb-cluster0", helmChartReleaseName)
+	admin0 := fmt.Sprintf("%s-0", admin)
+	defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+	sourceDatabaseChartName := testlib.StartDatabase(t, namespaceName, admin0, &helm.Options{
+		SetValues: map[string]string{
+			// HotSnap is available in NuoDB version 6.0.2 and above
+			"nuodb.image.tag":                                            "6.0.2",
```

Comment:
How does this work with the image inject in Bamboo? I guess master and rel/6.0-dev and up will have supported product versions, but not rel/5.1-dev, etc.

Reply:
Good point! I forgot about Bamboo. I will skip the test based on version constraints.